### ISSUE TITLE
ULS UNITS - Fix CULU theme-level accessibility issues from SiteImprove

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/uls#490]
@manolobevia I'm starting to go through all of the dev sites in SiteImprove in prep to send them to CIT. There are a few issues that are coming up as theme-level that I am fixing on CULU and then once I'm through all of those (hopefully by the end of September), we can do another upstream to push all of these fixes out to all units at once.

Issues to fix (checked means it has been added to CULU theme, not to each unit):
- [x] Staff profiles default image needs `alt=""` - (Image with no alt attribute 1.1.1)
- [x] Staff profile cards each need unique landmark names - (Landmarks with identical names 1.3.1)
- [x] Full Hours area text not meeting color contrast ratio - (Color contrast is insufficient 1.4.3)
- [x] Skip link should be included in a landmark - (Content not included in landmarks 1.3.1)
- [ ] Attribute arial-label not allowed on element div at this point. From line 347, column 1; to line 347, column 100 - (HTML Validation - Annex)
- [ ] Bad value search-home for attribute type on element input. From line 420, column 1; to line 420, column 88 - (HTML Validation - Annex)
- [ ] No space between attributes. At line 494, column 31 - (HTML Validation - Annex)
- [ ] The frameborder attribute on the iframe element is obsolete. Use CSS instead. From line 530, column 37; to line 530, column 293 - (HTML Validation - Annex)
- [ ] The scrolling attribute on the iframe element is obsolete. Use CSS instead. From line 530, column 37; to line 530, column 293 - (HTML Validation - Annex)
- [ ] The marginheight attribute on the iframe element is obsolete. Use CSS instead. From line 530, column 37; to line 530, column 293 - (HTML Validation - Annex)
- [ ] The marginwidth attribute on the iframe element is obsolete. Use CSS instead. From line 530, column 37; to line 530, column 293 - (HTML Validation - Annex)
- [ ] End tag address seen, but there were open elements. From line 575, column 1; to line 575, column 10 - (HTML Validation - Annex)
- [ ] Unclosed element div. From line 566, column 1; to line 566, column 23 - (HTML Validation - Annex)
- [ ] Headings in any content block should be black #000, they currently default to gray.
- [ ] No option to skip content on Login or Lost Password pages

**Comment by manolobevia** [comment]
@kevinkidwell don't you want to wait until the content is added to all the uls sites, then check in siteimrprove? I am doing an accessibility audit on culu (https://github.com/cul-it/uls/issues/484). Please add any issue there. I am moving these fixes to #484

**Comment by manolobevia** [comment]
@kevinkidwell, I think it makes sense first to focus on accessibility issues on CULU, then downstream the changes to all the uls sites. For content, like I said before it makes sense to check if the content is accessible when all the content is added. I am planning to go through this audit in our next sprint.

**Comment by manolobevia** [comment]
and I will leave the HMTL and CSS validation for the end. @mbmartini @mhk33. Any news about validating HTML and CSS from CIT?

**Comment by mhk33** [comment]
I was basically told to do the best you can with validation. 100% is ideal, but I mentioned problems with having to fix Drupal contributed modules, and that making those changes will make updates more difficult, they may get overwritten on update, etc. Sean understands that 100% may not be possible.

**Comment by kevinkidwell** [comment]
@manolobevia Only sites that have all of their content migrated are being checked with this but many of the theme-level issues are going to be consistent across the board so we can go ahead and start addressing them. We need to start getting these sites to CIT so we can be fixing theme-level issues and content-level issues at the same time for those already live/fully migrated.

I'm going to be running through all of our live sites and those dev sites that are finished migrated content. All live or fully migrated sites are listed below along with their current SiteImprove scores.

Annex - 94.1
Fine Arts - 89.6
Vet - 93.7
PSL - 73.9
Hotel - 91.7
Makerspace - ? (currently being added to my account)
Engineering - 85.4
Math - 91.0
Music - 92.7

I don't mind editing on CULU for any issues that I run into. I already pushed a few fixes (listed above) to Dev and can keep doing that if you want or anything that I find theme-level I can just add to #484 for you to fix. Which would you prefer?

**Comment by manolobevia** [comment]
https://github.com/cul-it/uls/issues/646 will take care of the validation

**Comment by manolobevia** [comment]
https://github.com/cul-it/uls/issues/548 will take care of the issue with login page.
